pennylane/queuing.py

```diff
@@ -301,9 +301,9 @@ def stop_recording(cls):
         try:
             yield
         except Exception as e:
-            cls._active_contexts = previously_active_contexts
             raise e
```

Comment:
do we still need the `except` logic at all?

Reply:
nope! updated